src/Covid19Dashboard/PlotChart/index.jsx

```diff
@@ -7,12 +8,11 @@ import {
 import { numberWithCommas } from '../dataUtils.js';
 import './PlotChart.less';
 
-
 class CustomizedAxisTick extends React.Component {
   render() {
     const { x, y, payload } = this.props; // eslint-disable-line react/prop-types
-    const val = payload.value; // eslint-disable-line react/prop-types
-    const formattedDate = `${new Date(val).getUTCMonth() + 1}/${new Date(val).getUTCDate()}`;
+    const val = payload.value.replace(/-/g, '/'); // eslint-disable-line react/prop-types
```

Comment:
i think if you are using `moment` then you don't need all these `.replace(/-/g, '/')`